src/stacks-hierarchy/Root_LocationSearchByTextScreen/components/LocationSearchContent.tsx

```diff
@@ -141,6 +149,22 @@ export function LocationSearchContent({
             onAddFavorite={onAddFavorite}
           />
         )}
+        {onlyStopPlacesCheckboxEnabled && (
+          <CheckboxWithLabel
+            label={t(LocationSearchTexts.onlyStopPlacesCheckbox)}
+            checked={onlyStopPlaces}
+            onPress={(v) => {
```

Comment:
Not sure what `v` stands for here, `checked` might be a better name?

Reply:
Short answer:
`v` is short for value, and I think it is pretty common to use, much like using `i` for item or `ev` for event. 

Long answer:
I actually had named it `checked` originally, but it wasn't really precise since it isn't actually checked yet. So I changed it to `newChecked`, but felt it was unnecessarily verbose, so I just thought "f* it,  let's just use `v`" 💯💪

Reply:
Fair enough, though not great 😅